mundane-parse/src/main/scala/com/ambiata/mundane/parse/TextEscaping.scala

```diff
@@ -0,0 +1,86 @@
+package com.ambiata.mundane.parse
+
+import scalaz.Equal
+import scalaz.Scalaz._
+
+object TextEscaping {
+
+  val escapeChar = '\\'
```

Comment:
Can we make the `escapeChar` a parameter? Also the `escapeChar` is being used verbatim on L.28.